lib/controller/page/type/page.summary/page.summary.controller.js

```diff
@@ -130,7 +132,8 @@ SummaryController.attachEmailSubmissions = async (submissionId, userData, submis
       const subject = formatEmail(userSubject)
       const addAttachment = getInstanceProperty('service', 'attachUserSubmission') || false
       const pdfData = SummaryController.generatePDFPayload(userData, submissionId, 'user')
-      const userSubmission = generateEmail('user', from, subject, emailUrlStub, submissionId, addAttachment, pdfData)
+      const emailBody = composeEmailBody(userData, userData.contentLang, emailTypes.USER)
```

Comment:
I'd be tempted to make composeEmailBody a private function.
Can't think of any instance of us using it that's not directly related to generateEmail.

Reply:
That's true but generates email woild have to pass through lots of objects that it itself does not care about.

Reply:
I also think we should have it as a private method. We would only need to give one more argument (`userData`), `language` can be derived from that and `recipientType` is already in `generateEmail`.

Once this PR is in, we'll be able to remove a few of the arguments too

Reply:
With this current implementation, if someone wants to reuse the code, they have to know two functions and how to construct them, rather than just one.